WordPress/Classes/ViewRelated/Post/PostCardCell.xib

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="15400" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="16096" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES">
```

Comment:
Sometimes just by opening a `.xib` Xcode changes its code (well, tks Xcode).

While I don't think this is a huge problem, check those files before committing so we can keep the PRs as small as possible!

Reply:
ack, I already discarded those changes once, didn't see them slip in again!